### ISSUE TITLE
vertex/workbench-instance-stuck-in-provisioning doesn't work correctly

### ISSUE THREAD

**Issue by adamlc** [GoogleCloudPlatform/gcpdiag#100]
After trying to debug why a Vertex AI workbench isn't working the GCP docs pointed me to this tool. Unfortunately this particular runback doesn't seem to work with the following command, I have tried in both Cloud Console and Locally.

```shell
./gcpdiag runbook vertex/workbench-instance-stuck-in-provisioning \
    --project="***" \
    --parameter "instance_name=***" \
    --parameter "zone=europe-west2-a"
```

When running I get the following error

```
gcpdiag 🩺 0.72

Starting runbook inspection [Alpha Release]
project: ***, parameters: {instance_name=***,project_id=***,zone=europe-west2-a}

[ERROR] vertex/workbench-instance-stuck-in-provisioning: Diagnostic Tree has not been registered or does not exist. Please verify the product and class name, and ensure the file containing the class does not have any syntax error
```

Am I doing something wrong? The [GCP docs](https://cloud.google.com/vertex-ai/docs/general/troubleshooting-workbench?component=any#provisioning-stuck-gcpdiag) have exactly the same command (which actually has a missing slash on the end of the first line).

**Comment by schweikert** [comment]
Hi Adam, thanks for reporting this. We had to do a rollback of the most recent version (0.73) because of problems with the docker image, and I think that that's the reason why it is not working now. We will work on soon publishing a new fixed version.

**Comment by ropeck** [comment]
Hello, thanks for reaching out with this error. I'm happy to share that we have rolled out a fix to the docker image issue with version 0.74 which is now live and available on the cloud shell as referenced in the GCP docs. 

If you are still seeing this error, please let me know and I can follow up right away.

**Comment by adamlc** [comment]
Thank you both for jumping on this quickly!

I can confirm that 0.74 fixes the issue!
